ROS2/ROS_workspace/cli_app/src/main.cpp

```diff
@@ -0,0 +1,77 @@
+// main.cpp
+#include <iostream>
+#include <string>
+#include <thread>
+#include <atomic>
+#include <opencv2/opencv.hpp>
+#include <mutex>
+#include <chrono>
+//
+// Forward declarations
+void launchGUI();
+void printCLIOptions(const std::string& arg);
+void printCLICommands();
+void handleCommand(const std::string& command);
+void printColoredAsciiImage();
+void printIntroduction();
```

Comment:
Should we show the CLI options and commands after the introduction art?

Reply:
This is done if you enter the command "help", enter a wrong command, or launch with "-h" as an argument